.drone.yml

```diff
@@ -47,5 +46,36 @@ steps:
         include:
           - master
       event:
+        include:
+          - push
+          - tag
         exclude:
           - pull_request
+
+  - name: gen-manifests
+    image: golang:latest
+    commands:
+      - wget https://github.com/kubernetes-sigs/kustomize/releases/download/v1.0.11/kustomize_1.0.11_linux_amd64
```

Comment:
nit, i'd probably add in `-O /usr/local/bin/kustomize` to skip line 60

Reply:
Added